src/linux/backend-v4l2.cpp

```diff
@@ -1362,6 +1397,11 @@ namespace librealsense
                                 // Drop partial and overflow frames (assumes D4XX metadata only)
                                 bool partial_frame = (!compressed_format && (buf.bytesused < buffer->get_full_length() - MAX_META_DATA_SIZE));
                                 bool overflow_frame = (buf.bytesused ==  buffer->get_length_frame_only() + MAX_META_DATA_SIZE);
+                                if (_dev.buf_type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE) {
+                                    /* metadata size is one line of profile, temporary disable validation */
```

Comment:
Could you implement a better solution for that, that will still detect partial frame and overflow frame?

Reply:
in ipu6 case we will always have full frame as it padding it by itself.
We can work on this in future